src/util.js

```diff
@@ -43,6 +43,10 @@ export const recursiveMerge = (a, b) => {
     return ret;
 };
 
+export const uniquify = (values) => {
+    return [...new Set(values)];
```

Comment:
I'm not sure we guarantee that people have a `Set` polyfill, so I don't think this will work?